lmdeploy/pytorch/check_env/__init__.py

```diff
@@ -76,6 +80,14 @@ def __check_transformers_version():
         try:
             import transformers
             trans_version = version.parse(transformers.__version__)
+            min_version = version.parse(MIN_TRANSFORMERS_VERSION)
+            max_version = version.parse(MAX_TRANSFORMERS_VERSION)
+            if trans_version < min_version or trans_version > max_version:
+                logger.warning('LMDeploy requires transformers version: '
```

Comment:
logger = get_logger('lmdeploy')  should be created in this func

Reply:
Why